apple/Sources/Sargon/Extensions/Methods/Prelude/Decimal192+Wrap+Functions.swift

```diff
@@ -123,19 +123,11 @@ extension Decimal192 {
 extension Decimal192 {
 
 	private func rounded(decimalPlaces: UInt8, roundingMode: RoundingMode) -> Self {
-		precondition(
-			decimalPlaces <= Decimal192.maxDivisibility,
-			"Decimal places MUST be 0...18, was: \(decimalPlaces)"
```

Comment:
Should I remove this from Kotlin too?

Reply:
I just removed it because it lowered code coverage - we panic in Rust anyway, so not needed to have it there.

Reply:
but we still should improve rounding methods a bit :) can do it later. create some throwing and some non throwing (using mode `towardsZero` never fails). And perhaps even make number of places an enum?